helper.js

```diff
@@ -210,9 +210,9 @@ HELPER.compile = (() => {
 						console.warn( warn.toString() );
 				});
 
-			Fs.writeFileSync( file, prefixed.css );
+				Fs.writeFileSync( file, prefixed.css );
 
-			HELPER.log.success(`Autoprefixed file ${ Chalk.yellow( file ) }`);
+				HELPER.log.success(`Autoprefixed file ${ Chalk.yellow( file ) }`);
```

Comment:
could be safer access to have `HELPER.log('success', 'message')` instead, making method a single level to HELPER